VENDORS.md

```diff
@@ -24,67 +25,64 @@ If you notice any mistakes, or would like to contribute, feel free to **make** a
+| [Ringer Keys](https://ringerkeys.com/)                       | [x]      | [x]  | [x]   |        |         | [x]       | Greater Boston, MA         |
+| [Space Cables](https://spaceholdings.net/)                   |          | [x]  | [x]   | [x]    | [x]     |           | Austin, TX                 |
 | [Spaztik Cables](https://spaztikcables.com/)                 |          |      |       | [x]    |         |           | Utah                       |
 | [Switch Couture](https://switchcouture.com/)                 |          |      |       |        |         | [x]       | Florida                    |
 | [Switchmod](https://switchmod.net/)                          |          | [x]  |       |        |         |           | Minnesota -- Massachusetts |
-| [SwitchTop](https://switchtop.com/)                          | [x]      |      | [x]   |        |         | [x]       | ?                          |
+| [SwitchTop](https://switchtop.com/)                          | [x]      |      | [x]   |        | [x]     |           | Texas                      |
 | [Teal Technik](https://tealtechnik.com/)                     | [x]      | [x]  | [x]   |        |         |           | ?                          |
 | [Tez Cables](https://tezcables.com/)                         |          |      |       | [x]    |         |           | ?                          |
-| [TheKey.Company](https://thekey.company/)                    | [x]      | [x]  | [x]   | [x]    | [x]     | [x]       | Michigan                   |
-| [Thockeys](https://thockeys.com/)                            | [x]      |      | [x]   | [x]    | [x]     |           | Westminster, MA            |
-| [Thockpop](https://thockpop.com/)                            | [x]      | [x]  | [x]   | [x]    | [x]     |           | Houston, TX                |
+| [TheKey.Company](https://thekey.company/)                    | [x]      | [x]  | [x]   |        | [x]     | [x]       | Michigan                   |
+| [Thockeys](https://thockeys.com/)                            | [x]      | [x]  | [x]   |        | [x]     |           | Westminster, MA            |
+| [Thockpop](https://thockpop.com/)                            | [x]      | [x]  | [x]   |        | [x]     |           | Houston, TX                |
 | [Typr Tools](https://typr.tools/)                            | [x]      | [x]  | [x]   |        |         |           | California (Northern)      |
 | [Upgrade Keyboards](https://upgradekeyboards.com/)           | [x]      | [x]  | [x]   | [x]    | [x]     | [x]       | Houston, TX                |
-| [WASD Keyboards](https://wasdkeyboards.com/)                 |          |      |       |        | [x]     | [x]       | California                 |
+| [WASD Keyboards](https://wasdkeyboards.com/)                 | [x]      |      |       |        | [x]     | [x]       | California                 |
```

Comment:
they also offer cables

Reply:
ooh good catch

Reply:
[updated](https://github.com/zhol0777/is-this-good/pull/11/commits/e5e59855dd1fd389304f45e8f5833ceda9e79cbe)